app/src/main/java/com/github/mik629/android/fundamentals/data/db/models/MovieWithActorsAndGenres.kt

```diff
@@ -0,0 +1,41 @@
+package com.github.mik629.android.fundamentals.data.db.models
+
+import androidx.room.Embedded
+import androidx.room.Junction
+import androidx.room.Relation
+import com.github.mik629.android.fundamentals.domain.model.Movie
+import com.github.mik629.android.fundamentals.domain.model.MovieDetails
+
+class MovieWithActorsAndGenres(
+    @Embedded
+    val movieEntity: MovieDbEntity,
+    @Relation(
+        parentColumn = MovieActorCrossRef.COLUMN_MOVIE_ID,
```

Comment:
parentColumn = MovieActorCrossRef.COLUMN_MOVIE_ID,
entityColumn = MovieActorCrossRef.COLUMN_ACTOR_ID,

are right?
or should be one column from MovieDbEntity and other from ActorDbEntity

Reply:
As far as I understand, we bind entity ids in crossRef (via foreign keys) while here we say which column is a key and which are the values.       
P.S. I've tried your suggestion and it didn't work